### PR TITLE
Fix few typos in user guide

### PR DESCRIPTION
**Pull request by jozefizso**

### CODE REVIEW

**Comment by yazeng** [comment]
Can one of the admins verify this patch?

**Comment by msftgits** [comment]
Hi, I am closing and re-opening this PR to bump the CLA bot. Sorry for the inconvenience!

File: docs/PTFUserGuide.md [hunk 1]
```diff
@@ -246,19 +246,19 @@ For example, if a user has the following configuration in the PTFconfig
 
 __Examples__
 
-1.   Command “set ptfprop” can be used in the script to display all properties in the PTFconfig. User will get the outputs like:
+1.   Command “set ptfprop" can be used in the script to display all properties in the PTFconfig. User will get the outputs like:
```
Comment (yazeng) [inline docs/PTFUserGuide.md]: The first quotation mark should be changed too.

File: docs/PTFUserGuide.md [hunk 2]
```diff
@@ -246,19 +246,19 @@ For example, if a user has the following configuration in the PTFconfig
...
 
 ```
 2008-04-07 14:46:20.121 [Comment] STDOUT: ptfpropfeaturename=MS-ATSVC
 2008-04-07 14:46:20.121 [Comment] STDOUT: ptfproptestname=ATSVC
 2008-04-07 14:46:20.121 [Comment] STDOUT: ptfpropversion=2.0
 2008-04-07 14:46:20.121 [Comment] STDOUT: ptfpropservername=MS-Sever
 ```
-2.  User can get the property value and use it directly in the script via the environment variables with “ptfprop” prefix.
+2.  User can get the property value and use it directly in the script via the environment variables with “ptfprop" prefix.
```
Comment (yazeng) [inline docs/PTFUserGuide.md]: The first quotation mark should be changed too.

File: docs/PTFUserGuide.md [hunk 3]
```diff
@@ -364,11 +364,11 @@ Configure <Adapters> section of .ptfconfig file like this:
 
 * "xsi:type": The value of this attribute must be "powershell".
 * "name": The value of this attribute is the name of the adapter interface which will be bound to the PowerShell adapter proxy.
-* “scriptdir”: The value of this attribute is the path of the directory which contains the corresponding PowerShell scripts.
+* “scriptdir": The value of this attribute is the path of the directory which contains the corresponding PowerShell scripts.
```
Comment (yazeng) [inline docs/PTFUserGuide.md]: The first quotation mark should be changed too.

File: docs/PTFUserGuide.md [hunk 4]
```diff
@@ -364,11 +364,11 @@ Configure <Adapters> section of .ptfconfig file like this:
...
 
 __Parameters__
 
-Parameters are passed from PowerShell adapter to PowerShell script directly so that users can use them without any parsing. Users can also set values to out/ref parameters directly in their PowerShell scripts and the PowerShell adapter will pass them back to the test suite. The parameter name is the same as defined in the adapter interface. For example, If a parameter “int a” is defined, then you can use $a in the PowerShell adapter directly.
+Parameters are passed from PowerShell adapter to PowerShell script directly so that users can use them without any parsing. Users can also set values to out/ref parameters directly in their PowerShell scripts and the PowerShell adapter will pass them back to the test suite. The parameter name is the same as defined in the adapter interface. For example, If a parameter “int a" is defined, then you can use $a in the PowerShell adapter directly.
```
Comment (yazeng) [inline docs/PTFUserGuide.md]: The first quotation mark should be changed too.

File: docs/PTFUserGuide.md [hunk 5]
```diff
@@ -412,14 +412,14 @@ public void TestPowerShellAdapter()
...
-## <a name="4.4"/>Extensive Logging Support 
+## Extensive Logging Support 
 
 PTF provides problem-oriented logging capabilities (begin/end test group, verification pass/failure, requirement capture, debugging). Most of the logging is done automatically so that the test suite developer does not have to write them manually.
 
 * Test Asserting failure/success,
 * Entering/Exiting Adapter Code.
 
-PTF provides the ability to define “logging profiles” in PTF configuration files. These profiles provide a way to define which logging event is piped to which sink. Logging profiles can be set dynamically by testing code. 
+PTF provides the ability to define “logging profiles" in PTF configuration files. These profiles provide a way to define which logging event is piped to which sink. Logging profiles can be set dynamically by testing code.
```
Comment (yazeng) [inline docs/PTFUserGuide.md]: The first quotation mark should be changed too.

File: docs/PTFUserGuide.md (hunk 1 above)
Comment (yazeng) [inline docs/PTFUserGuide.md]: The first quotation mark should be changed too.
There're also many other places that need to replace the first quotation mark.

**Comment by yazeng** [comment]
Hi Jozef,
Thank you for reviewing user guide for us and find us so many typos ;)
The overall changes look good to me except one comment.

Thanks again,
Yan

File: docs/PTFUserGuide.md (hunk 1 above)
Comment (jozefizso) [inline docs/PTFUserGuide.md]: I've fixed the missing incorrect quotations.

**Comment by yazeng** [comment]
Looks good to me. Thank you!!

**Pull request merged**